litt/disktable/disk_table.go

```diff
@@ -665,7 +736,16 @@ func (d *DiskTable) Flush() error {
 	return nil
 }
 
-func (d *DiskTable) SetCacheSize(_ uint64) error {
+func (d *DiskTable) SetWriteCacheSize(size uint64) error {
+	if ok, err := d.fatalErrorHandler.IsOk(); !ok {
+		return fmt.Errorf("Cannot process SetCacheSize() request, DB is in panicked state due to error: %w", err)
```

Comment:
method name is `SetWriteCacheSize`, not `SetCacheSize`

Reply:
fixed